grouper/api/routes.py

```diff
@@ -19,17 +20,19 @@
 from grouper.handlers.health_check import HealthCheck
 
 HANDLERS = [
-    (r"/users", Users),
-    (r"/users/{}".format(NAME_VALIDATION), Users),
-    (r"/token/validate".format(NAME_VALIDATION), TokenValidate),
-    (r"/public-keys", UsersPublicKeys),
+    (r"/debug/health", HealthCheck),
+    (r"/grants", Grants),
+    (r"/grants/{}".format(PERMISSION_VALIDATION), Grants),
     (r"/groups", Groups),
     (r"/groups/{}".format(NAME_VALIDATION), Groups),
     (r"/permissions", Permissions),
     (r"/permissions/{}".format(PERMISSION_VALIDATION), Permissions),
+    (r"/public-keys", UsersPublicKeys),
     (r"/service_accounts", ServiceAccounts),
     (r"/service_accounts/{}".format(NAME_VALIDATION), ServiceAccounts),
+    (r"/users", Users),
+    (r"/users/{}".format(NAME_VALIDATION), Users),
     (r"/multi/users", MultiUsers),
-    (r"/debug/health", HealthCheck),
+    (r"/token/validate".format(NAME_VALIDATION), TokenValidate),
```

Comment:
Not from this diff, but does this `format` do anything?

Reply:
Good catch. I'll add a diff on to this PR that gets rid of that.